pkg/network/network.go

```diff
@@ -396,7 +396,7 @@ func NameForPortNumber(svc *corev1.Service, portNumber int32) (string, error) {
 }
 
 // PortNumberForName resolves a given name to a portNumber as defined by an EndpointSubset.
-func PortNumberForName(sub corev1.EndpointSubset, portName string) (int32, error) {
+func PortNumberForName(sub *corev1.EndpointSubset, portName string) (int32, error) {
```

Comment:
Actually passing by value is better for performance here.

Reply:
Done